src/components/Ticket/TicketModal.tsx

```diff
@@ -0,0 +1,43 @@
+import { Overlay } from '../actionBar/pnrSearch/PnrSearch.styled';
+import { Ticket } from './Ticket';
+import FullScreenLoader from '../FullScreenLoader/FullScreenLoader';
+import CloseIcon from '@mui/icons-material/Close';
+import { useGetTicketData } from './utils/useGetTicketData';
+import { useSearchParams } from 'react-router-dom';
+
+export const TicketModal = ({ cancelModal }: { cancelModal: () => void }) => {
+    const [searchParams, setSearchParams] = useSearchParams();
+    const pnrNumber: string | null = searchParams.get('pnr');
```

Comment:
may be pnr as enum?

Reply:
Since pnr wasn't used for passing to components, I could avoid assigning a type
Resolved in a7ece662e28ede42e328db75978d2d9d9560689c